scrapy/exporters.py

```diff
@@ -144,8 +144,10 @@ def _export_xml_field(self, name, serialized_value):
         elif is_listlike(serialized_value):
             for value in serialized_value:
                 self._export_xml_field('value', value)
-        else:
+        elif isinstance(serialized_value, six.text_type):
```

Comment:
Hey @curita -- it doesn't need to check bytes anymore because we've changed the BaseExporter class to send text/unicode instead of bytes as it was before.